### PR TITLE
Species info hex

### PR DESCRIPTION
**Pull request by AlexOn1ine**
Please review throughly

### CODE REVIEW

File: include/pokemon.h [hunk 1]
```diff
@@ -443,17 +443,17 @@ struct SpeciesInfo /*0x8C*/
             u32 isFrontierBanned:1; // This species is not allowed to participate in Battle Frontier facilities.
             u32 padding4:14;
             // Move Data
- /* 0x80 */ const struct LevelUpMove *levelUpLearnset;
- /* 0x84 */ const u16 *teachableLearnset;
-            const u16 *eggMoveLearnset;
- /* 0x88 */ const struct Evolution *evolutions;
- /* 0x84 */ const u16 *formSpeciesIdTable;
- /* 0x84 */ const struct FormChange *formChangeTable;
+ /* 0x7E */ const struct LevelUpMove *levelUpLearnset;
```
Comment (DizzyEggg) [inline include/pokemon.h:446]: Pointers need to be 4-bit aligned, ending with 0, 4, 8 or C.

**Review by DizzyEggg — CHANGES_REQUESTED** [review]

File: include/pokemon.h (hunk 1 above)
Comment (AlexOn1ine) [reply include/pokemon.h]: fixed hopefully

**Comment by AlexOn1ine** [comment]
ready for rereview

File: include/pokemon.h [hunk 2]
```diff
@@ -445,15 +445,15 @@ struct SpeciesInfo /*0x8C*/
...
  /* 0x80 */ const struct LevelUpMove *levelUpLearnset;
  /* 0x84 */ const u16 *teachableLearnset;
-            const u16 *eggMoveLearnset;
- /* 0x88 */ const struct Evolution *evolutions;
- /* 0x84 */ const u16 *formSpeciesIdTable;
- /* 0x84 */ const struct FormChange *formChangeTable;
+ /* 0x88 */ const u16 *eggMoveLearnset;
+ /* 0x8C */ const struct Evolution *evolutions;
+ /* 0x90 */ const u16 *formSpeciesIdTable;
+ /* 0x94 */ const struct FormChange *formChangeTable;
```
Comment (DizzyEggg) [inline include/pokemon.h:451]: Why does it go from 0x94 to 0xB4? It should be
```
/* 0x98 */ struct ObjectEventGraphicsInfo overworldData;
```
Not sure what's the length of the `ObjectEventGraphicsInfo`, but the next entry should be `0x98 + struct size`

**Review by DizzyEggg — CHANGES_REQUESTED** [review]

File: include/pokemon.h (hunk 2 above)
Comment (AlexOn1ine) [reply include/pokemon.h:451]: the size is 20 bytes. should be fixed

**Review by DizzyEggg — APPROVED** [review]

**Pull request merged**